lib/cli.js

```diff
@@ -302,13 +316,42 @@ function execute(programArgs) {
         debug: Boolean(cli.debug)
     };
 
+    userConfig.plugins = userConfig.plugins || [];
+    for (const plugin of cli.plugin) {
+        userConfig.plugins.push(plugin);
+    }
+
+    userConfig.rules = userConfig.rules || {};
+    for (const rule of cli.rule) {
+        let offset = rule.indexOf(":");
+
+        if (offset !== -1) {
+            let key = rule.substring(0, offset);
+            let value = rule.substring(offset + 1, rule.length);
+            try {
+                value = JSON.parse(rule.substring(offset + 1, rule.length));
+            } catch (e) {
+                errorReporter.reportFatal(
+                    `There was an error trying to parse '${value}': ${e.message}`
+                );
+            }
+
+            userConfig.rules[key] = value;
+            continue;
+        }
+
+        userConfig.rules[rule] = true;
```

Comment:
If I'm to understand correctly, when we reach this line it will set a rule to `true`. So our userConfig might look like:
```
{
  "rules": {
    "my-rule": true
  }
}
```

Boolean as rule value was the format of solium v0 (which is now deprecated).
Please change this to `userConfig.rules[rule] = "error";`

This means that by default any rule that a user specifies (without the `: <config>` part) is treated as an `error` by solium.